samples/bluetooth/direct_test_mode/README.rst

```diff
@@ -115,9 +115,73 @@ The following antenna switching patterns are possible:
 The application supports a maximum of 19 antennas in the direction finding mode.
 The RADIO can control up to 8 GPIO pins for the purpose of controlling the external antenna switches used in direction finding.
 
-The antenna is chosen by writing consecutive numbers to the SWITCHPATTERN register.
-This means that the antenna GPIO pins act like 8-bit registers.
-In other words, for the first antenna, antenna pin 1 is active, for the second antenna, pin 2 is active, for the third antenna, pins 1 and 2 are active, and so on.
+Antenna matrix configuration
+----------------------------
+
+To use this sample to test the Bluetooth Direction Finding feature, additional configuration of GPIOs is required to control the antenna array.
+An example of such configuration is provided in a devicetree overlay file :file:`nrf5340dk_nrf5340_cpunet.overlay`.
+
+The overlay file provides the information about of the GPIOs to be used by the Radio peripheral to switch between antenna patches during the Constant Tone Extension reception or transmission.
+At least one GPIO must be provided to enable antenna switching.
+
+The GPIOs are used by the Radio peripheral in order given by the ``dfegpio#-gpios`` properties.
+The order is important because it affects the mapping of the antenna switching patterns to GPIOs (see `Antenna patterns`_).
+
+To test Direction Finding, provide the following data related to antenna matrix design:
+
+* GPIO pins to ``dfegpio#-gpios`` properties in the :file:`nrf5340dk_nrf5340_cpunet.overlay` file.
+* The default antenna to be used to receive the PDU ``dfe-pdu-antenna`` property in the :file:`nrf5340dk_nrf5340_cpunet.overlay` file.
+
+.. note::
+   The PDU antenna is also used for the Reference period transmission and reception.
```

Comment:
Do we need to capitalize "Reference" here?

Reply:
We don't need. Fixed